healthcheck/settings/base.py

```diff
@@ -32,6 +32,14 @@
     "phonenumber_field",
     "celery",
     "django_filters",
+    # monitoring apps
+    "django_prometheus",
+    "health_check",
+    "health_check.db",
+    "health_check.cache",
+    "health_check.storage",
+    #    'health_check.contrib.celery',
+    "health_check.contrib.redis",
```

Comment:
In production, we're not using Redis for celery, we're using RabbitMQ. Will this break if we're not using Redis as the celery backend?

Reply:
![image](https://user-images.githubusercontent.com/18465274/91712593-b99b4480-eb90-11ea-8ed7-7ebbd42e49b5.png)
It will look like this but the rabbitmq will be available and redis will be not